### PR TITLE
Fix a bug where a unicode response are not properly displayed in `DocService`

### PR DESCRIPTION
**Pull request by ikhoon**
Motivation:

A Unicode response such as `Hello, \uD83D\uDC4B` should be rendered to `Hello, 👋`.
However, the `Response.text()` of fetch API returns an escaped string itself.

Modifications:

- Use `JSONbig.parse()` and `JSONbig.stringify()` to get an unescaped string.

Result:

You no longer see escaped Unicode from a JSON response in `DocService`

### CODE REVIEW

**Review by minwoox — APPROVED** [review]
Thanks!
Don't we need to handle Thrift?

**Comment by ikhoon** [comment]
> Don't we need to handle Thrift?

Thrift worked without any modifications.

File: docs-client/src/lib/transports/annotated-http.ts [hunk 1]
```diff
@@ -112,16 +112,18 @@ export default class AnnotatedHttpTransport extends Transport {
...
     });
     const applicationType = httpResponse.headers.get('content-type') || '';
-    const response = await httpResponse.text();
-    if (response.length > 0) {
-      if (applicationType.indexOf('json') > -1) {
-        const prettified = prettify(response);
-        if (prettified.length === 0) {
-          return response;
-        }
+    if (applicationType.indexOf('json') > -1) {
```
Comment (trustin) [inline docs-client/src/lib/transports/annotated-http.ts:115]: nit: `>= 0`?

File: docs-client/src/lib/transports/annotated-http.ts [hunk 2]
```diff
@@ -112,16 +112,18 @@ export default class AnnotatedHttpTransport extends Transport {
...
+      const prettified = jsonPrettify(
+        JSON.stringify(await httpResponse.json()),
```
Comment (trustin) [inline docs-client/src/lib/transports/annotated-http.ts:117]: Could you make sure this does not break the case where a user sends a property with a big integer? e.g.

```
{
  "big_int": 2934812984902318490
}
```

Could you make sure that:

- a response with a large value is parsed correctly and displayed in the response pane;
- a request with a large value is sent as-is to the server;
- a request with a large value is minified correctly into the browser location field; and
- the request shared via a URL is deserialized and prettified correctly into the debug form?

Comment (trustin) [reply docs-client/src/lib/transports/annotated-http.ts:117]: We probably need front-end testing automation :sob:

Comment (ikhoon) [reply docs-client/src/lib/transports/annotated-http.ts:117]: > Could you make sure this does not break the case where a user sends a property with a big integer? e.g.

Sorry, I missed this comment. 😭 

> We probably need front-end testing automation 😭

Indeed, we need it. 😭

Comment (ikhoon) [reply docs-client/src/lib/transports/annotated-http.ts:117]: I didn't know that `JSON.parse("...")` does not properly handle a large value.
We might need to patch `jsonPrettify` to render Unicode.

Comment (ikhoon) [reply docs-client/src/lib/transports/annotated-http.ts]: - [x] a response with a large value is parsed correctly and displayed in the response pane;
- [x] a request with a large value is sent as-is to the server;
- [x] a request with a large value is minified correctly into the browser location field; and
- [x] the request shared via a URL is deserialized and prettified correctly into the debug form?

I found out that [json-bigint](https://www.npmjs.com/package/json-bigint) keeps large numbers from truncating.
Finally, all problems are solved. 😀

**Review by trustin — APPROVED** [review]
Thanks, @ikhoon !

**Review by minwoox — COMMENTED** [review]
Thanks!

**Pull request merged**